src/utils/encoreLogOutTimer.js

```diff
@@ -55,16 +59,40 @@ function EncoreLogOutTimer() {
     let timeTillLogOut = (time > 0) ? time : 0;
 
     // Only for testing. If the function is run for tests, set the timeout no longer than 2 seconds
-    if (isTest && timeTillLogOut > 2000) {
-      timeTillLogOut = 2000;
+    if (isTest && timeTillLogOut > 8000) {
```

Comment:
Adjusted for a longer time here to let another test for deleting "nyplIdentityPatron" cookie to run first.
See test/Header.test.js, line 273.